Automattic/sensei
# Merge master into feature/exporter

### Changes proposed in this Pull Request

* Update feature branch with changes from master.
* Add changes from #3446, #3455 to exporter

### Testing instructions

* Make sure merged exporter functionality still works - #3413 #3412 #3409
* Make sure #3446, #3455 works in exporter
